public/data/Hojung/StoryData.json

```
@@ -0,0 +1,17 @@
+[
+  {
+    "src": "/images/Hojung/profile_img.jpeg",
+    "writer": "_yum_s",
+    "writerTime": "16분전"
```

```suggestion
    "created_at": "16분전"
```
이렇게는 어떨까요? 일반적으로 서버에서 넘어오는 데이터의 키값은 snake_case입니다.

---

수정완료 했습니다